docs/content/monitoring.md

```diff
@@ -5,9 +5,7 @@ weight: 30
 restrictedtoc: true
 ---
 
-## Monitoring
```

Comment:
Found this looking a bit silly, updated:
![image](https://user-images.githubusercontent.com/870638/130416692-4c304d36-4e43-4dfb-bcde-73bc59a21d9b.png)
⬇️ 
![image](https://user-images.githubusercontent.com/870638/130416738-3fe326e0-4f61-4a3d-8a32-0e4ceae6e6a5.png)